cypress/utils/plugins/security-analytics-dashboards-plugin/constants.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const TWENTY_SECONDS_TIMEOUT = { timeout: 20000 };
+
+export const INDICES = {
+  DETECTORS_INDEX: '.opensearch-detectors-config',
+  PRE_PACKAGED_RULES_INDEX: '.opensearch-pre-packaged-rules-config',
+  CUSTOM_RULES_INDEX: '.opensearch-custom-rules-config',
+};
+
+export const PLUGIN_NAME = 'opensearch_security_analytics_dashboards';
+export const BASE_API_PATH = '/_plugins/_security_analytics';
+
+export const NODE_API = {
```

Comment:
do you need all these or just the `RULES_BASE`

Reply:
We plan to add commands, for use by other test cases, that will use those API; so I'd like to leave them for now.